internal/app/server/handler_test.go

```diff
@@ -248,11 +333,10 @@ func TestGetParcel(t *testing.T) {
 
 			w := httptest.NewRecorder()
 			body := strings.NewReader("")
-			r := httptest.NewRequest(http.MethodGet, "/api/v1/parcel/"+tc.parcelID, body)
-			r = mux.SetURLVars(r, map[string]string{"id": tc.parcelID})
+			r := httptest.NewRequest(http.MethodGet, "/api/v1/parcel/"+strconv.Itoa(tc.status)+strconv.Itoa(tc.offset)+strconv.Itoa(tc.limit), body)
```

Comment:
I think "strconv.Itoa(tc.status)+strconv.Itoa(tc.offset)+strconv.Itoa(tc.limit" this part has no meaning here. It passes the wrong url like '/api/v1/parcel/1/20/20' . This is not our endpoint.

Reply:
better to build the url string

Reply:
done